meson_options.txt

```diff
@@ -34,6 +34,8 @@ option('debug-shell', type : 'string', value : '/bin/sh',
        description : 'path to debug shell binary')
 option('debug-tty', type : 'string', value : '/dev/tty9',
        description : 'specify the tty device for debug shell')
+option('extra-debug', type : 'string',
+       description : 'enable extra debugging (hashmap,mmap-cache)')
```

Comment:
I don't think so... Consistency is worthwhile if you're switching between one system and the other, but I think most people do either one or the other. If anything, this would inconvenience people who configured with previous name or have scripts which use the old name.